ui/src/components/songsView/SongsLyrics.tsx

```diff
@@ -0,0 +1,250 @@
+import { Box, Chip, Grid, Stack, Typography, useMediaQuery } from '@mui/material';
+import { SongViewSchema } from '../../types/song.types';
+import { flatMusicKeysOptions, sharpMusicKeysOptions } from '../../constants';
+import { ReactNode, useCallback, useEffect, useState } from 'react';
+
+interface SongsLyricsProps {
+  chordStatus: boolean;
+  changeKey: number;
+  song: SongViewSchema | undefined;
+  split: number;
+  useFlat: boolean;
+}
+
+const SongsLyrics = ({ chordStatus, changeKey, song, split, useFlat }: SongsLyricsProps) => {
+  const isDesktop = useMediaQuery('(min-width:768px)');
+  const noSplit = isDesktop ? split : 1;
+  const [finalLyrics, setFinalLyrics] = useState<ReactNode[]>();
+
+  const countParagraph = (inputSong: SongViewSchema | undefined) => {
+    const lyricsLine = inputSong?.chordLyrics.split('\n');
+    let para = 0;
+    lyricsLine &&
+      lyricsLine.map((line) => (line.includes('{') && line.includes('}') ? para++ : null));
+  };
+
+  
+
+  const getColor = (label:any) => {
+    const regexPattern = /[A-G][#b]?(m)?/;
+    label.match(regexPattern)
+    
+    if (label === 'C') {
```

Comment:
can u put this as getColor function as like a constant file? so have a json where you define the chords with colors. so example is find the key 'C' to get the value '#FAFABE' in the json. avoid using these many `if` statements.

Reply:
yes, I am moving it to the ui/constants.ts